libs/sdk-backend-spi/src/workspace/attributes/index.ts

```diff
@@ -64,4 +68,11 @@ export interface IWorkspaceAttributesService {
      * @returns promise returning batch of attribute references.
      */
     getCommonAttributesBatch(attributesRefsBatch: ObjRef[][]): Promise<ObjRef[][]>;
+
+    /**
+     * Get information about the given attribute's dataset
+     * @param ref - ref of the attribute
+     * @returns promise of metadata object
+     */
+    getAttributeDatasetMeta(ref: ObjRef): Promise<IMetadataObject>;
```

Comment:
Do we have some more concrete type for the returned value? Also, to keep naming consistent, we should probably drop the "Meta" part, the other methods here also return metadata objects.